index_configuration_tool/main.py

```diff
@@ -53,11 +69,11 @@ def validate_plugin_config(config: dict, key: str):
         raise ValueError("Invalid auth configuration (Password without user) for endpoint: " + supported_endpoint[0])
 
 
-def validate_logstash_config(config: dict):
-    if "input" not in config or "output" not in config:
-        raise ValueError("Missing input or output data from Logstash config")
-    validate_plugin_config(config, "input")
-    validate_plugin_config(config, "output")
+def validate_pipeline_config(config: dict):
+    if SOURCE_KEY not in config or SINK_KEY not in config:
+        raise ValueError("Missing source or sink configuration in Data Prepper pipeline YAML")
```

Comment:
Minor point, but might be nice to make this two conditions so you can specify exactly which one is missing. Not a blocker.

Reply:
Incorporated in https://github.com/opensearch-project/opensearch-migrations/pull/218